pyproject.toml

```diff
@@ -18,7 +18,7 @@ python-dateutil = "^2.8.1"
 whitenoise = "^5.0.1"
 uuid = "^1.30"
 faker = "^8.11.0"
-django-postgres-setfield = "^0.0.1"
+django-postgres-setfield = {git = "https://github.com/benperlman/django-postgres-setfield.git"}
```

Comment:
Why are we using this fork?

Reply:
In my commit comments I explained why:

> Needed to update "django-postgres-setfield" as well to a custom
> version that fixed a known issue with 'ugettext_lazy'. See:
> https://code.djangoproject.com/ticket/30165.

Basically, without this change, we'd need to change the model to get code to work.  In my comments I also noted that fact.

https://github.com/sfbrigade/intentional-walk-server/pull/175/commits/be18f43f5b43b6213d7f677a0092ff53143f5ed3

Reply:
I've read the comments and seen __what__ it fixes but I was asking that to hear more about the justification (**why**).

I'm not saying we should use this fork, but if we were to, we should be using a fork maintained on /sfbrigade/ rather than a personal fork. 

My thought process and issue is because it adds maintenance overhead of a private fork. We're now needing to update the fork once in a while for security updates as well as this repo, and it may not be clear each time.

Reply:
If we do choose to go this route (not suggesting/or recommending we should), we should at least have it tied to a specific commit/release or even nested as a submodule fixed under a commit hash under `github.com/sfbrigrade/django-postgres-setfield.git`